packages/playground/blueprints/src/lib/steps/activate-plugin.ts

```diff
@@ -65,22 +64,61 @@ export const activatePlugin: StepHandler<ActivatePluginStep> = async (
 				}
 			}
 
-			if ( null === $response ) {
-				die('Plugin activated successfully');
-			} else if ( is_wp_error( $response ) ) {
-				throw new Exception( $response->get_error_message() );
+			if (is_wp_error($response)) {
```

Comment:
What if $response is `null` or `false`?

Reply:
When using `throw` a WP_Error caused by the plugin activation would cause PHP to fail with a 255 response code and we wouldn't be able to read the error message AFAIK.

We would also need to use `try/catch` for the activation code which would be less elegant, especially in a case where we don't consider all WP_Errors created by the the activation process to be errors for us. 

More details about why we don't consider all WP_Errors to be activation errors are documented here https://github.com/WordPress/wordpress-playground/pull/2066/files/aaf683d47f401d18f9958a8437b7abe4f428d915#diff-4a02b123bb6311fff3ec089ca1ade75c75f1af35aaa4f1a717f14d444fc9b485R81